bigstream/transform.py

```diff
@@ -716,8 +716,6 @@ def displacement_field_composition_square_root(
 
     # loop over scale levels
     level_values = zip(iterations, shrink_spacings, smooth_sigmas)
-    logger.info(f'Displacement vector leveled iterations: {level_values}')
```

Comment:
as is now the logger is not very harmful but not helping either because it does not display the zipped "content". But if anybody changes it to display the values - it is bad because it consumes the zip iterator and for will not execute so it is safer to remove it

Reply:
I fixed this in a different PR.
It has always annoyed me, and I always forget, that range and zip do not return lists or tuples, but instead iterator objects whose print methods are not very helpful. I wrapped the zip with a tuple() and that fixes things. You see the print that you want and the set of values are still available for iteration.